packages/react/src/combobox/ComboboxRenderItem.tsx

```diff
@@ -26,59 +27,78 @@ export function ComboboxRenderItem({
 	secondaryText,
 	tertiaryText,
 }: ComboboxRenderItemProps) {
+	// deprecation warnings
+	if (process.env.NODE_ENV !== 'production' && inputValue) {
+		console.warn('Combobox: The `inputValue` prop is now unused.');
+	}
```

Comment:
I do wonder if these `console.warn`s are a source of dev mode slowdowns. Particularly for teams who may have lots of deprecated props and have accumulated a lot of these warnings. I was thinking we should remove the check via the `if` from `Box`, too. Anywho, just some thinking out loud for now, not a request for a change.